src/v3/test/integration/__snapshots__/authenticator-verification-email.test.tsx.snap

```diff
@@ -620,7 +620,6 @@ exports[`authenticator-verification-email renders correct form renders the otp c
                           id="credentials.passcode"
                           name="credentials.passcode"
                           type="text"
-                          value=""
```

Comment:
Is this expected?

Reply:
Ah yes, what was happening before this PR was when you would enter an invalid OTP code, it would reload the view but would clear the input field (even though visibly, the text was still left in the field). This was invalid and did not match parity (also was the root of our issues with the "field cannot be blank" error. Now since we reload the view with the same text (when there is an error) the field is not artificially cleared allowing the user to amend the original text/value.